source/_integrations/xbox.markdown

```diff
@@ -0,0 +1,50 @@
+---
+title: Xbox
+description: Instructions on how to set up Xbox devices in Home Assistant.
+ha_category:
+  - Media Player
+ha_iot_class: Cloud Polling
+ha_release: 0.117
+ha_codeowners:
+  - '@hunterjm'
+ha_domain: xbox
+---
+
+The Xbox integration allows you to control Xbox consoles from Home Assistant.
+
+Home Assistant authenticates with Xbox Live through OAuth2 using the Home Assistant account linking service. Set up the integration through **Configuration -> Integrations -> Xbox**.
```

Comment:
I wasn't sure what to do here because I plan to have this available through cloud linking, so didn't want to force users through the below steps, but it doesn't look like anything else that uses account linking mentions it in the docs?

Reply:
Looks good to me! Both ways should be documented.

Reply:
What's the status for the account linking? This isn't available yet?

Reply:
I’ve sent @balloob the info. Haven’t heard anything yet.

Reply:
So maybe we shouldn't mention this until that is ready?

Reply:
Sorry. By “I haven’t heard anything”, I meant I haven’t heard about it being done. He did confirm that it should be no problem to get done and requested the steps to get the client credentials.

Reply:
Ok, but then we're blocking this PR until confirmation that it's done. I was thinking it's better to unblock the PR and make another PR when it's ready.

Reply:
Circled up with Paulus and he said he would do it on Monday.

Reply:
@MartinHjelmare - Account Linking is now configured and verified working.